Makefile

```diff
@@ -5,12 +5,16 @@ IMAGE_TAG      ?= master
 IMAGE_REGISTRY ?= docker.io
 IMAGE_REPO     ?= kedacore
 
+BUNDLE_VERSION ?= 2.0.0
+# Default bundle image tag
+BUNDLE_IMG ?= $(IMAGE_REGISTRY)/$(IMAGE_REPO)/keda-olm-operator-bundle:$(BUNDLE_VERSION)
+
 IMAGE_CONTROLLER = $(IMAGE_REGISTRY)/$(IMAGE_REPO)/keda-olm-operator:$(IMAGE_TAG)
 
-ARCH       ?=amd64
-CGO        ?=0
-TARGET_OS  ?=linux
-VERSION ?= v2
+ARCH		?= amd64
+CGO			?= 0
+TARGET_OS	?= linux
+VERSION 	?= v2
```

Comment:
IMAGE_TAG and VERSION should be merged into one property, to VERSION, look here for an example  https://github.com/kedacore/keda/blob/a9e118f76d846a87d19d224a858009e7650f2fc5/Makefile#L4-L9